src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
+                  title="remove background"
+                  v-if="team.background"
+                  @click="removeBackground"
+                  outline
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="trashIcon" />
+                </ion-chip>
+
+                <ion-chip
+                  v-else
+                  @click="selectBackground"
+                  title="upload background"
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="uploadIcon" />
+                </ion-chip>
+                <inline-link-list showTitle :items="footerLinks">
+                  <li v-if="canEdit">
+                    <ion-button
+                      @click="intendEditFooterLinks"
+                      size="small"
+                      fill="clear"
+                    >
+                      <ion-icon size="small" :icon="editIcon" />
+                    </ion-button>
+                  </li>
+                </inline-link-list>
+              </div>
+            </ion-col>
+          </div>
+
+          <ion-toolbar>
+            <!-- Scrollable Segment -->
+            <ion-segment
+              scrollable
+              value="about"
+              mode="md"
+              @ionChange="segmentChanged($event)"
+            >
+              <ion-segment-button value="qrcode">
+                <ion-icon :icon="qrCodeIcon" />
+              </ion-segment-button>
+              <ion-segment-button value="about">
+                <ion-label>About</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="news">
+                <ion-label>News</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="feed">
+                <ion-label>Feed</ion-label>
+              </ion-segment-button>
+            </ion-segment>
+          </ion-toolbar>
+
+          <div style="display: block; overflow-y: auto; height: 60vh">
+            <!-- Div About -->
+            <div v-if="state == 'about'">
               <div data-cy="description">
-                <render-md  adminMd :source="info" />
-                <ion-button data-cy-role="editModal" v-if="canEdit" color="light" @click="intendEditInfo" size="small" fill="clear">
-                  Edit Text
-                </ion-button>
+                <h2 data-cy="title" class="subTitle">
+                  {{ team.name }}
+                </h2>
+                <p>
+                  {{ team.info }}
+                  <ion-button
+                    data-cy-role="editModal"
+                    v-if="canEdit"
+                    color="primary"
+                    @click="intendEditInfo"
+                    size="small"
+                    fill="clear"
+                  >
+                    Edit Text
+                  </ion-button>
+                </p>
+                <render-md adminMd :source="info" />
```

Comment:
you have this, but also `{{ team.info }}` above. this doesn't make any sense. I think you want to keep this one (but at the position above) and fix it as
```suggestion
                <render-md adminMd :source="team.info" />
```